azure-webapp-maven-plugin/README.md

```diff
@@ -1,182 +1,175 @@
 # Maven Plugin for Azure App Service
+
+## Table of Content
+  - [Overview](#overview)
+  - [Prerequisites](#prerequisites)
+  - [Quick Start](#quick-start)
+  - [Goals](#goals)
+      - [`azure-webapp:config`](#azure-webappconfig)
+      - [`azure-webapp:deploy`](#azure-webappdeploy)
+  - [Configuration Details](#configuration-details)
+      - [`<javaVersion>`](#javaversion)
```

Comment:
shall we organize the items in alphabetical order？

Reply:
I think do not need organize them in alphabetical order